src/Verifier/verifier.go

```diff
@@ -0,0 +1,144 @@
+package Verifier
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+	"slices"
+	"strings"
+)
+
+// This is for alternative names for blips e.g. CSharp, CS, C# all should be counted as C#)
+// var alt_names_str = `Go;golang,go-lang
+// 					C++;cpp
+// 					C#;csharp,cs
+// 					Python;python3,py
+// 					`
+
+// Map of alternative names for the same blip
+var alt_names = make(map[string]string)//{"golang":"Go","go-lang:Go","cpp":"C++","csharp":"C#","cs":"C#","python3":"Python","py":"Python"}
+
+// Checks that the given string matches the defined header of the specfile
+func checkHeader(header string) bool {
+	correctHeader := "name,ring,quadrant,isNew,moved,description"
+	return header == correctHeader
+}
+
+// The pattern is created as a singleton so that we don't need 
+// to compile a new on every line that is checked
+var regexPattern *regexp.Regexp = nil
+
+// Creates the regex pattern string with the names on the rings
+func createRegexPattern(ring1, ring2, ring3, ring4 string) {
+	var err error
+	regexPattern, err = regexp.Compile(fmt.Sprintf("^(([^,\n])([^,\n])*),([%s]%s|[%s]%s|[%s]%s|[%s]%s),([Dd]ata management|[Dd]atastore|[Ii]nfrastructure|[Ll]anguage),(false|true),-?[0123],(([^,\n])([^,\n])*)",
```

Comment:
This does seem rather hard to maintain. While ok for now, I think it will be a problem that the quadrant names are hardcoded in the regex as they could change.